index.html

```diff
@@ -1,24 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="wrapper">
-        <h1 class="page-title">Awesome books</h1>
-        <div class="book-list">
-        </div>
-        <form action="./index.html" method="POST" name="book-form">
-          <input type="text" placeholder="Title" name="title" aria-label="fill in book title"><br>
-          <input type="text" placeholder="Author" name="author" aria-label="fill in book author's name"><br>
-          <input type="submit" value="Add" aria-label="Submit the form" class="submitBtn">
-        </form>
+  <div class="wrapper">
+    <nav class="navbar">
+      <ul class="navlist">
+        <li class="list active"><a href='#' class="">List</a></li>
+        <li class="add-new"><a href='#'>Add new</a></li>
+        <li class="contact"><a href='#'>Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="list-books">
+      <h1 class="page-title">Awesome books</h1>
+      <div class="book-list">
       </div>
-        <script src="script.js"></script>
-      
-</body>
+    </section>
+
+    <section class="add-books hide"> 
+      <h1 class="page-title">Add a new book</h1> 
+      <form action="./index.html" method="POST" name="book-form">
+        <input type="text" placeholder="Title" name="title" aria-label="fill in book title"><br>
+        <input type="text" placeholder="Author" name="author" aria-label="fill in book author's name"><br>
+        <input type="submit" value="Add" aria-label="Submit the form" class="submitBtn">
+      </form>
+    </section>
+
+
+    <section class="contact-info hide">
+      <h1 class="page-title">Contact information</h1>
+      <p class="tagline">Do you have any questions or you just want to say "Hello"?</p>
+      <p class="tagline">You can reach out to us!</p>
+      <ul class="list">
+        <li class="list-item">Our Email: <a href="mailto:someone@yoursite.com">mail@mail.com</a></li>
+        <li class="list-item">Our phone number:03986563564636</li>
+        <li class="list-item">Our Address:<address>Plo 2201, Lower Konge</address>  </li>
+      </ul>
+    </section>
+
+    <footer>
+      <p>Copyright 2022, All rights reserved</p>
+    </footer>
+
+  </div>
+  <script src="script.js"></script>
+  </body>
```

Comment:
- [ ] As per the project requirements, I think you should add the current date below the navigation bar.

> For this requirement, you should use a library - [Luxon](https://moment.github.io/luxon/#/install).
> See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

 **Wireframe design:**

![t1](https://user-images.githubusercontent.com/70483835/165645322-55fd98ea-93d6-4d22-91d1-3ed7e3b54cc5.png)